deploy/OSD-6246-flowcontrol-fix/config.yaml

```
@@ -6,4 +6,4 @@ selectorSyncSet:
     values: ["4.5","4.6"]
   - key: hive.openshift.io/version-major-minor-patch
     operator: NotIn
-    values: ["4.6.13"]
+    values: ["4.6.13","4.6.15","4.6.16"]
```

So we don't have to keep updating this, let's just knock out a reasonably large swath of possible patch numbers (13 to 99):

```suggestion
    values: ["4.6.13","4.6.14","4.6.15","4.6.16","4.6.17","4.6.18","4.6.19","4.6.20","4.6.21","4.6.22","4.6.23","4.6.24","4.6.25","4.6.26","4.6.27","4.6.28","4.6.29","4.6.30","4.6.31","4.6.32","4.6.33","4.6.34","4.6.35","4.6.36","4.6.37","4.6.38","4.6.39","4.6.40","4.6.41","4.6.42","4.6.43","4.6.44","4.6.45","4.6.46","4.6.47","4.6.48","4.6.49","4.6.50","4.6.51","4.6.52","4.6.53","4.6.54","4.6.55","4.6.56","4.6.57","4.6.58","4.6.59","4.6.60","4.6.61","4.6.62","4.6.63","4.6.64","4.6.65","4.6.66","4.6.67","4.6.68","4.6.69","4.6.70","4.6.71","4.6.72","4.6.73","4.6.74","4.6.75","4.6.76","4.6.77","4.6.78","4.6.79","4.6.80","4.6.81","4.6.82","4.6.83","4.6.84","4.6.85","4.6.86","4.6.87","4.6.88","4.6.89","4.6.90","4.6.91","4.6.92","4.6.93","4.6.94","4.6.95","4.6.96","4.6.97","4.6.98","4.6.99"]
```